services/app/src/contexts/ChessBoardContext.jsx

```diff
@@ -21,15 +23,41 @@ const ChessBoardContextProvider = ({ children }) => {
   const [score, setScore] = useState(0);
   const [globalScore, setGlobalScore] = useState(0);
 
+  const currentPlayer = useMemo(
+    () => players.find((user) => user.id === currentPlayerId),
+    [players],
+  );
+
+  const voters = useMemo(() => {
```

Comment:
this should work:
  const sortedPlayers = useMemo(() => {
    const otherPlayers = players.filter(p => p.id !== currentPlayerId);
    return [currentPlayer, ...otherPlayers];
  }, []);
  const voters = useMemo(() => sortedPlayers.filter(p => p.role === PLAYER_ROLES.VOTER), [sortedPlayers]);
  const spectators = useMemo(() => sortedPlayers.filter(p => p.role === PLAYER_ROLES.SPECTATOR), [sortedPlayers]);